http-server/src/test/java/xtdb/jackson/TxDeserializerTest.java

```diff
@@ -0,0 +1,63 @@
+package xtdb.jackson;
+
+import clojure.java.api.Clojure;
+import clojure.lang.IFn;
+import clojure.lang.Keyword;
+import clojure.lang.PersistentHashMap;
+import clojure.lang.PersistentVector;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.module.SimpleDeserializers;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.junit.jupiter.api.Test;
+import xtdb.tx.Ops;
+import xtdb.tx.Put;
+import xtdb.tx.Tx;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TxDeserializerTest {
+
+    private final ObjectMapper objectMapper;
+
+    public TxDeserializerTest() {
+        ObjectMapper objectMapper = new ObjectMapper();
+        SimpleModule module = new SimpleModule("JsonLd-test");
```

Comment:
I'm not a fan that these don't test the actual code - let's see what it takes to get the module itself into Java?

Reply:
Do you want me to move all mappers to Java?

Reply:
I would prefer the mappers to stay in `jackson.clj` to quickly add stuff as it comes up.

Reply:
I can see why this is preferable for development turnaround time in the short-term; long-term, we likely won't be adding things as frequently. With that in mind, maybe once we've bottomed it out, we make the migration?